sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/src/Generated/Models/SecuritySettings.cs

```diff
@@ -8,7 +8,7 @@
 namespace Azure.ResourceManager.RecoveryServices.Models
 {
     /// <summary> Security Settings of the vault. </summary>
-    internal partial class SecuritySettings
+    public partial class SecuritySettings
```

Comment:
`SecuritySettings` is too general, please rename it to a more specific name, such as prepending RP name to it by adding it to `prepend-rp-prefix` in [autorest.md](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/src/autorest.md)